docs/README.md

```diff
@@ -178,108 +178,122 @@ Refer to your editor's documentation to learn how to connect to an LSP server.
 
 ## Rules
 
-Rules are individual type checks that detect common issues in your code—such as incompatible assignments, missing imports, or invalid type annotations. Each rule focuses on a specific pattern and can be turned on or off depending on your project’s needs.
-See [rules] for a reference of all supported rules.
+Rules are individual checks that ty performs to detect common issues in your code — such as
```

Comment:
grammar nit: I think this em-dash should just be a comma

Reply:
I love em dashes.
```suggestion
Rules are individual checks that ty performs to detect common issues in your code, such as
```